Pipe siad's log to disk.

ro-tex opened the pull request:
Closes #25

ro-tex committed:
Pipe siad's log to disk.

ro-tex commented on scripts/run.sh:
```diff
@@ -15,4 +15,6 @@ END
 # occur.
-exec $SIAD_CMD "$@"
+exec $SIAD_CMD "$@" \
+  1>"$SIA_DATA_DIR/siad_stdout.log" \
+  2>"$SIA_DATA_DIR/siad_errout.log"
```
Hm, would it make more sense to append to those files or just keep the latest log?

ro-tex replied:
This kind of resolved itself with the request to keep outputting the logs to `stdout` and `stderr`. We can't keep them between executions, so anyone who wants that will have to do it themselves.

ro-tex committed:
Keep outputting the data to `stdout` and `stderr`.
Make sure the dir exists.

ro-tex committed:
Prepare the log files, so we don't get stale output or errors from `tail`.

ro-tex committed:
Clean up.

interstella-5555 reviewed (commented):
Personally I don't see a benefit in copying output log to files if it is already available - see https://stackoverflow.com/a/41147654
It is just sia in docker container and I would keep it as simple as possible and as familiar as possible to people used to run their services with docker. 

On a side note, there is a way to run the container with logging to local file in docker docs https://docs.docker.com/config/containers/logging/local/

ro-tex committed:
Merge branch 'master' into ivo/logs

ro-tex committed:
Remove the logs on disk logic, add info to the README.

ro-tex commented:
Good point, @kwypchlo. There's a lot about Docker I don't know, thanks for helping out!

I removed the scripted log output and added a section to the README. I'd appreciate any input about that as well.
